bin/steps/pip-install

```diff
@@ -38,10 +38,10 @@ if [ ! "$SKIP_PIP_INSTALL" ]; then
         mcount "failure.none-version"
     fi
 
-    if grep -q 'django==1.*' requirements.txt; then
+    if grep -qi '^django==1.*' requirements.txt; then
```

Comment:
Does `requirements.txt` accept case insensitive filenames?

Reply:
pip install is case insensitive 👍